### PR TITLE
Add babel, css ,index.html and js file

### PR DESCRIPTION
**Pull request by eyobyoseph**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<title>Fork Me! FCC: Test Suite Template</title>
+		<link rel="stylesheet" href="css/style.css" />
+	</head>
+
+	<body>
+		<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (eyobyoseph) [inline index.html:10]: I recommend you put a proper indentation since proper indentation makes your code readable and clean. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (eyobyoseph) [inline index.html:23]: I would like to advise you to avoid adding unnecessary comments to your code. Such kinds of things decrease your code readability. 😁

File: index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+		<script src="js/index.js"></script>
+		<h1>Hello World!</h1>
+		<h2>Hello World, we are software developers!</h2>
```
Comment (eyobyoseph) [inline index.html:26]: Please try not to include an unrelated line of code. 💯

File: index.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+				<img
+					src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+					id="image"
+				/>
+				<p id="img-caption">
+					Dr. Norman Borlaug, third from the left, trains biologists
+					in Mexico on how to increase wheat yields - part of his
+					life-long war on hunger.
+				</p>
+			</div>
```
Comment (eyobyoseph) [inline index.html:39]: I would like to advise you to put these elements in the header tag. Since the header tag is a semantic element, it clearly describes the content of the section. 👍

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	width: auto;
+	height: auto;
+}
+#tribute-info {
+	width: 800px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
```
Comment (eyobyoseph) [inline css/style.css:33]: Great job you have done it. 💯 
I would like to advise you to give one line space after each css selectors.

**Review by eyobyoseph — COMMENTED** [review]
STATUS: CHANGES REQUIRED ♻️ :
Hi [@eyobyoseph](https://github.com/eyobyoseph) ,

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

image
 thankyou very much

**Review by Owusu-Desmond — APPROVED** [review]
Hi @eyobyoseph  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Good to use GitHub inline feature
📌 Great working on the setup of the project ✅
📌 Good use of inline comments
📌 Nice collaboration with student

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
